cdc/sink/mysql.go

```diff
@@ -114,6 +112,15 @@ type mysqlSink struct {
 	cancel         func()
 }
 
+func needSwitchDB(ddl *model.DDLEvent) bool {
+	if len(ddl.TableInfo.Schema) > 0 {
```

Comment:
the old logic is `shouldSwitchDB := len(ddl.TableInfo.Schema) > 0 && ddl.Type != timodel.ActionCreateSchema`

when `len(ddl.TableInfo.Schema)` returns `0`, `shouldSwitchDB` should be `false`

Reply:
Can you add a test for it?